codigo/casoPractico/src/casopractico/CasoPractico.java

```diff
@@ -0,0 +1,258 @@
+                    "Ingrese 2 para crear una cuenta en Twitter\n"+
+                    "Ingrese 3 para crear una cuenta en Whatsapp\n"+
+                    "Ingrese 4 para crear una cuenta en Telegram\n"+
+                    "Ingrese 5 para crear una cuenta en Signal\n"+
+                    "Ingrese 6 para crear una cuenta en Instagram\n"+
+                    "Ingrese 7 para crear una cuenta en Flickr\n");
+            opcion = entrada.nextInt();
+            if(opcion == 1){
+                facebook = crearFacebook();
+            }else{
+                if(opcion == 2){
+                    crearTwitter();
+                }else{
+                    if(opcion == 3){
+                        whatsapp = crearWhatsapp();
+                    }else{
+                        if(opcion == 4){
+                            crearTelegram();
+                        }else{
+                            if(opcion == 5){
+                                signal = crearSignal();
+                            }else{
+                                if(opcion == 6){
+                                    crearInstagram();
+                                }else{
+                                    if(opcion == 7){
+                                        flickr = crearFlickr();
+                                    }else{
+                                        System.out.println("Opcion incorrecta");
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            if ((opcion >= 1) && (opcion <= 7)){
+                contador = contador + 1;
+            }
+            entrada.nextLine();
+            System.out.println("Ponga si para dejar de crear cuentas");
+            sa = entrada.nextLine();
+            salida = sa.toLowerCase();
+            if(salida.equals("si")){
+                bandera = false;
+            }
+        }
+        System.out.println(facebook + whatsapp + signal + flickr);
+        String cadena = obtenerMensaje(contador);
+        System.out.println(cadena);
+    }
+    public static String crearFacebook(){
+        Scanner entrada = new Scanner(System.in);
+        String usuario;
+        int edad;
+        String ciudad;
+        String pais;
+        String correo;
+        String cadena = "";
+        System.out.println("Ingrese el nombre de usuario");
+        usuario = entrada.nextLine();
+        System.out.println("Ingrese su edad");
+        edad = entrada.nextInt();
+        entrada.nextLine();
+        System.out.println("Ingrese el nombre de su ciudad");
+        ciudad = entrada.nextLine();
+        System.out.println("Ingrese el nombre de su pais");
+        pais = entrada.nextLine();
+        System.out.println("Ingrese su correo");
+        correo = entrada.nextLine();
+        cadena = String.format("%s\tFacebook\nUsuario: %s\nEdad: %d\n"
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.